specification/compute/resource-manager/Microsoft.Compute/ComputeRP/stable/2023-03-01/examples/dedicatedHostExamples/DedicatedHost_ListAvailableSizes.json

```diff
@@ -0,0 +1,19 @@
+{
+  "parameters": {
+    "subscriptionId": "{subscriptionId}",
+    "resourceGroupName": "myResourceGroup",
+    "api-version": "2023-03-01",
+    "hostGroupName": "myDedicatedHostGroup",
+    "hostName": "myHost"
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "value": [
+          "Dsv3-Type1",
```

Comment:
During resize, we only change the hwGen and not the vmFamily type, so we should say something like "Dsv3-Type2 and Dsv3-Type3"